### PR TITLE
Improve plotting

### LINKED ISSUE #143: Improve plotting
Some functions already exist in plot_utils, but following the rework of the metrics we should add more different ways of plotting the metrics results, and think about which one are interesting to the user.

### PR DESCRIPTION
**Pull request by thuiop**
Adding some utility functions for plotting the metrics results. It uses seaborn which should thus be installed. You can have a look in the intro notebook ; it is not finished yet. Currently, it plots:
- Distribution of MSR, PSNR, SSIM, IOU (if applicable)
- For each target measure, a scatter plot with true/detected measures, and an errorbar plot for the difference
- A joint plot (scatter + histograms) for MSR against blendedness
- The efficiency matrices

If someone as any idea of additional plot which should be supported, they are welcome.

Closes #143.

### CODE REVIEW

**Comment by thuiop** [comment]
This PR has been open for long enough ; it should be ready for review. Everything is not perfect right now, notably on having a coherent front with the old plotting functions and some documentation could be added, but I will do this in another PR.

**Comment by ismael-mendoza** [comment]
Hi @thuiop, sorry I think the review slipped my mind. I will review it today

File: notebooks/scarlet-measure.ipynb [hunk 1]
```diff
@@ -3,10 +3,19 @@
   {
```
Comment (ismael-mendoza) [inline notebooks/scarlet-measure.ipynb:3]: <p>Did you mean to remove the output of this cell? either it doesn't matter users can run the notebook and get the output, just wanted to check if this was your intention. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSSTDESC/BlendingToolKit/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (thuiop) [reply notebooks/scarlet-measure.ipynb:3]: <p>I must have forgotten to run it again ; I also have to add the interactive=True back so I will do it now.</p>

**Comment by ismael-mendoza** [comment]
everything looks great now, thanks @thuiop !

**Pull request merged**